lib/OpenQA/WebSockets.pm

```diff
@@ -43,7 +43,6 @@ sub startup {
     my $ca = $r->under('/')->to('Auth#check');
     $ca->get('/' => {json => {name => $self->defaults('appname')}});
     my $api = $ca->any('/api');
-    $api->get('/is_worker_connected/<worker_id:num>')->to('API#is_worker_connected');
```

Comment:
Just double-checking... but is it okay to simply remove that route?

Reply:
Yes. This is not part of the "official REST API". The route is only part of the internal IPC between the services. Since it is no longer needed I'm removing it.